From 4420a0d96f6fbc0e8955a10134709006868cd73f Mon Sep 17 00:00:00 2001
From: Bai Ping <b51503@freescale.com>
Date: Wed, 1 Apr 2015 23:46:15 +0800
Subject: [PATCH 0611/1594] MLK-10532-01 ARM: dts: add device node for tempmon
 module

commit 224204c8a1a0994f0c0dacd19d009fd6ea6b98b7 from
git://git.freescale.com/imx/linux-2.6-imx.git

add tempmon module device node used by thermal driver for i.MX7D.

Signed-off-by: Bai Ping <b51503@freescale.com>
---
 arch/arm/boot/dts/imx7d.dtsi |    9 ++++++++-
 1 files changed, 8 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 023a704..d3d0742 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -568,7 +568,6 @@
 				compatible = "syscon";
 				reg = <0x30350000 0x10000>;
 				clocks = <&clks IMX7D_CLK_DUMMY>;
-				status = "disabled";
 			};
 
 			anatop: anatop@30360000 {
@@ -593,6 +592,14 @@
 				};
 			};
 
+			tempmon: tempmon {
+				compatible = "fsl,imx7d-tempmon";
+				interrupts = <GIC_SPI 49 IRQ_TYPE_LEVEL_HIGH>;
+				fsl,tempmon =<&anatop>;
+				fsl,tempmon-data = <&ocotp>;
+				clocks = <&clks IMX7D_PLL_SYS_MAIN_CLK>;
+			};
+
 			snvs: snvs@30370000 {
 				compatible = "fsl,sec-v4.0-mon", "simple-bus";
 				#address-cells = <1>;
-- 
1.7.5.4

